MovieNight/Alert.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Alert.swift
+//  MovieNight
+//
+//  Created by Tassia Serrao on 26/07/2017.
+//  Copyright © 2017 Tassia Serrao. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct Alert {
```

Comment:
Because otherwise, you would have repeated code everywhere. Really nice use of SRP (Single Responsibility Principle)